src/layout/msWorkspace/msWorkspace.ts

```diff
@@ -534,10 +536,10 @@ export class MsWorkspace extends WithSignals {
     //     }
     // }
 
-    setApps(apps) {
-        this.apps = apps;
-        this.categorizedAppCard._loadApps(apps);
-    }
+    // setApps(apps) {
+    //     this.apps = apps;
+    //     this.categorizedAppCard._loadApps(apps);
+    // }
```

Comment:
Unused?

Reply:
Yeah, wasn't sure if I should remove it completely or not.